src/kixi/datastore/metadatastore/dynamodb.clj

```diff
@@ -108,15 +108,15 @@
   [conn update-event]
   (info "Update: " update-event)
   (comment "This implementation is not idempotent, need a check to prevent repeat segement adds"
-    (db/append-list conn
-                    (primary-metadata-table (:profile conn))
-                    id-col
-                    (::md/id update-event)
-                    :add
-                    [::md/segmentations]
-                    (:kixi.group/id update-event))))
-
-(defmethod update-metadata-processor ::cs/file-metadata-sharing-updated
+           (db/append-list conn
+                           (primary-metadata-table (:profile conn))
+                           id-col
+                           (::md/id update-event)
+                           :add
+                           [::md/segmentations]
+                           (:kixi.group/id update-event))))
+
+(defn update-sharing!
```

Comment:
Lose the !, i don't think anything else uses it in this code base. Could be misleading.